docs/source/support/release-notes.rst

```diff
@@ -76,14 +83,68 @@ Java Bindings
 
   See `issue #1092 <https://github.com/digital-asset/daml/issues/1092>`__ for details.
 
+- Added :ref:`agreement text <daml-ref-agreements>` of contracts: `#1110 <https://github.com/digital-asset/daml/issues/1110>`__
+
+  - **Java Bindings**
+
+    - Added field ``Optional<String> agreementText`` to ``data.CreatedEvent``, to reflect the change in Ledger API.
+
+  - **Java Codegen**
+
+    - Added generated field ``Optional<String> TemplateName.Contract#agreementText``.
+    - Added generated static method ``TemplateName.Contract.fromCreatedEvent(CreatedEvent)``.
+      This is the preferred method to use for converting a ``CreatedEvent`` into a ``Contract``.
+    - Added generated static method ``TemplateName.Contract.fromIdAndRecord(String, Record, Optional<String>)``.
+      This method is useful for setting up tests, when you want to convert a ``Record`` into a contract without having to create a ``CreatedEvent`` first.
+    - Deprecated generated static method ``TemplateName.Contract.fromIdAndRecord(String, Record)`` in favor of the new static methods in the generated ``Contract`` classes.
+    - Changed the generated :ref:`decoder utility class <daml-codegen-java-decoder-class>` to use the new ``fromCreatedEvent`` method.
+    - Changed the return type of the ``getDecoder`` method in the generated decoder utility class from ``Optional<BiFunction<String, Record, Contract>>`` to ``Optional<Function<CreatedEvent, Contract>>``.
+
+  How to migrate:
+
+  - If you are manually constructing instances of ``data.CreatedEvent`` (e.g.: for testing), you need to add an ``Optional<String>`` value as constructor parameter for the ``agreementText`` field.
+  - All calls to ``Contract.fromIdAndRecord`` should be changed to ``Contract.fromCreatedEvent``.
```

Comment:
```suggestion
  - You should change all calls to ``Contract.fromIdAndRecord`` to ``Contract.fromCreatedEvent``.
```

Reply:
There's the passive voice again. Thank you.